lib/module.js

```diff
@@ -489,7 +489,9 @@ Module._initPaths = function() {
 
   var nodePath = process.env['NODE_PATH'];
   if (nodePath) {
-    paths = nodePath.split(path.delimiter).concat(paths);
+    paths = nodePath.split(path.delimiter).concat(paths).filter(function(path) {
+      return !!path
+    });
```

Comment:
Yeah it probably is, given the input is a string it should be quite obvious what it does.